Mobile Test Runner for Jama/Mobile Test Runner for Jama/ProjectListViewController.swift

```diff
@@ -10,9 +10,14 @@ import UIKit
 
 class ProjectListViewController: UIViewController {
 
+    var currentUser: UserModel = UserModel()
+    
+    @IBOutlet weak var tempLabel: UILabel!
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        if currentUser.firstName != nil {
```

Comment:
This is just temporary code but I changed it to remove the check. It is fine if the name being set to the empty string. I didn't even notice that the warning was happening, good catch. That is good to know about Strings or I guess non-optional types in general.